packages/components/src/components/Log/Log.js

```diff
@@ -122,6 +150,105 @@ export class LogContainer extends Component {
     return false;
   };
 
+  updateScrollButtonCoordinates = () => {
+    if (this.props.enableLogScrollButtons) {
+      const logRectangle = this.logRef.current?.getBoundingClientRect();
+      const logElementRight =
+        document.documentElement?.clientWidth - logRectangle.right;
+
+      const scrollButtonTop = Math.max(0, logRectangle.top);
+
+      const scrollButtonBottom = Math.max(
+        0,
+        document.documentElement?.clientHeight - logRectangle.bottom
+      );
+
+      this.updateCssStyleProperty(
+        logElementRight,
+        Object.keys({ logElementRight })[0]
```

Comment:
this will always yield the string `logElementRight`, can we use that directly here, or even use the CSS custom property name instead so we don't have to transform between formats later?

similar for the other values

Reply:
removed `Object.keys` and passed a literal string as requested... however ass css custom properties have to start by `--` and js variables cannot start by these 2 characters so I kept the variable in the js file in camelCase and kept the transformation to kebab-case.

Reply:
While it's true that variable names cannot start with `-`, field names can. There should be no issue with something like `--log-element-right` as a field of the parent object as you're currently setting it.

![image](https://user-images.githubusercontent.com/2829095/126665539-13e83e29-b061-4454-8a1d-7299b66ba670.png)

Reply:
changed as requested